teacher.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './person_class'
+
+class Student < Person
```

Comment:
- I suppose you meant `class Teacher < Person`. kindly update this.

Reply:
- After correcting the class name, let's create the setter method for the specialization of the teacher like this
`  attr_accessor :specialization`